File: src/medium/3dprinter.go
package main

import (
	"fmt"
	"math"
)

func main() {
	var n int
	fmt.Scan(&n)
	d := 1.0
	for math.Pow(2, d-1) < float64(n) {
		d += 1
	}
	fmt.Println(d)
}

File: src/medium/3dprinter.java
import java.util.Scanner;

class ThreeDPrintedStatues {
    public static void main(String[] args) {
        Scanner s = new Scanner(System.in);
        int n = s.nextInt();
        int d = 1;
        while (Math.pow(2, d - 1) < n) {
            d += 1;
        }
        System.out.println(d);
    }
}
